modules/solid_mechanics/doc/content/source/materials/FluxBasedStrainIncrement.md

```diff
@@ -1 +1,31 @@
-!template load file=stubs/moose_object.md.template name=FluxBasedStrainIncrement syntax=/Materials/FluxBasedStrainIncrement
\ No newline at end of file
+# FluxBasedStrainIncrement
+
+!syntax description /Materials/FluxBasedStrainIncrement
+
+The strain is computed from flux instead of the displacements. The gradient of the flux is
+first computed from the gradient of each components of the flux specified by the user:
+
+!equation
+\nabla F = (\nabla F_x, \nabla F_y, \nabla F_z)
+
+where:
+
+- $F_x$ is the x-component of the flux specified with the [!param](/Materials/FluxBasedStrainIncrement/xflux) parameter
+- $F_y$ is the y-component of the flux, optionally specified with the [!param](/Materials/FluxBasedStrainIncrement/yflux) parameter
+- $F_z$ is the z-component of the flux, optionally specified with the [!param](/Materials/FluxBasedStrainIncrement/zflux) parameter
+
+The strain increment tensor is then computed as:
+
+!equation
+\boldsymbol{\epsilon}_{inc} = -\dfrac{1}{2}(\nabla F + ^T(\nabla F)) *  (1.0 - G) * _dt;
```

Comment:
`_dt` at the end of the equation makes the timestep size look weird due to subscripting the "d".